graphql_api/tests/test_bundle_analysis_measurements.py

```diff
@@ -111,46 +111,48 @@ def test_bundle_report_measurements(self, get_storage_service):
                     repository(name: $repo) {
                         ... on Repository {
                             commit(id: $commit) {
-                                bundleAnalysisReport {
-                                    __typename
-                                    ... on BundleAnalysisReport {
-                                        bundle(name: "super") {
-                                            name
-                                            measurements(
-                                                filters: $filters
-                                                orderingDirection: $orderingDirection
-                                                after: $after
-                                                interval: $interval
-                                                before: $before
-                                            ){
-                                                assetType
+                                bundleAnalysis {
```

Comment:
For tests where I overwrote the query in the test, what I went for [here](https://github.com/codecov/codecov-api/pull/842/files#r1785289848) was made a `_legacy.py` copy of the original file that I can then delete in the cleanup step. That way the old fields and the new fields still have a test lying around until the cleanup is done. (I think I interpret this change correctly but lmk if I missed where the original one is still around)